bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -347,9 +347,9 @@ public String getLabel(Widget w) {
                                         labelMappedOption = labelMappedOption.substring(0,
                                                 labelMappedOption.indexOf("[") + 1) + formatPatternOption + "]";
                                     } catch (IllegalArgumentException e) {
-                                        logger.warn(
-                                                "Exception while formatting value '{}' of item {} with format '{}': {}",
-                                                stateOption, itemName, formatPattern, e);
+                                        logger.debug(
+                                                "Mapping option value '{}' for item {} using format '{}' failed; mapping is ignored",
```

Comment:
For debugging purposes it would be good to at least include `e.getMessage()` in the log message in order to have a clue about the reason.

Reply:
I will add it.